src/tailwind/plugins/base/gc-base.js

```diff
@@ -6,7 +6,6 @@ module.exports = function () {
   return function ({ addBase, config }) {
     addBase({
       html: {
-        scrollBehavior: 'smooth',
```

Comment:
I'm removing smooth scrolling for now. On Gatsby's route change, if someone clicks on a link that's lower on a page, the route change would trigger a scrolling of the new page from that same lower point on the page to the top, which means it would trigger all the animations that depends on scroll position. Also, it's a lot of scrolling movement so probably not great for people who prefers reduced motion either. The only time I can see this being useful is if they click an anchor link on the same page. If we need that later, we'll see if we can add that just for that purpose.

Reply:
This makes sense to me! We encountered a similar bug that was resolved by removing `scrollBehavior: 'smooth'` on the Alumni Homesite 😄

Reply:
@rebeccahongsf Good to know. Yeah, I see that as a nice to have for same page anchor links, but I actually don't particularly like the scrolling on every route change. Too much movement 😂